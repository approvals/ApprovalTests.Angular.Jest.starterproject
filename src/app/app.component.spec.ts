import {describe, expect} from "@jest/globals";
import {printArray} from "approvals/lib//Utilities/Printers";
import {verify} from "approvals/lib/Providers/Jest/JestApprovals";

describe('ApprovalsStuff', () => {
  it('should approve', () => {
    expect("5").toBe("5");
  });

  it('should approve with approvals', () => {
    let actual = printArray("", ["Lada", "Pedro", "Llewellyn"]);
    expect(actual).toMatchSnapshot();
  });

  it('should approve with approvals for reals', () => {
    verify('foo');
  });
});
