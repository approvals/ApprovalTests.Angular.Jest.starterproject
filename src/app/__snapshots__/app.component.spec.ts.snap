// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApprovalsStuff should approve with approvals 1`] = `
"[0] => Lada
[1] => Pedro
[2] => Llewellyn
"
`;
